app/settings_strings.grdp

```diff
@@ -2576,20 +2576,20 @@
   <message name="IDS_SETTINGS_SITE_SETTINGS_JAVASCRIPT_BLOCKED_EXCEPTIONS" desc="Label for the blocked exceptions site list of the JavaScript content setting.">
     Not allowed to use JavaScript
   </message>
-  <message name="IDS_SETTINGS_SITE_SETTINGS_JAVASCRIPT_OPTIMIZER_ALLOWED" desc="Label for the enabled option of the JavaScript optimizer content setting.">
-    Sites can use the V8 optimizer
+  <message name="IDS_SETTINGS_SITE_SETTINGS_JAVASCRIPT_OPTIMIZER_ALLOWED" desc="Label for the enabled option of the JavaScript JIT content setting.">
```

Comment:
`This file is created by l10nUtil.js. Do not edit manually`

Reply:
I'm not really sure what to do about these, I don't think we normally edit the text of upstream strings except to replace Brave with Chrome. cc @mkarolin

Reply:
This feature might be exposed subsequently as a Brave Shields feature, so I think I might disable the string changes here, and implement a "disable-optimisation-is-jitless" flag in `about:flags` instead